### PR TITLE
Make component type names consistent

### PR DESCRIPTION
**Pull request by Eelviny**
### Step 1: Link to Jira issue
https://stackstate.atlassian.net/browse/STAC-13444

### Step 2: Description of changes
- Change formatting of all URNs reported by the agent to use a consistent style (changes included in stackpack also)
- Fix formatting for API Gateway Rest API

### Step 3: Did you add / update tests for your changes in the right area?
- [ ] Unit/Component test
- [ ] Integration test	

### Step 4: I'm confident that everything is properly tested:
I got a PO / QA Approval by:
- [ ] Name

### Step 5: Can we ship this feature to production?
- [ ] Yes, I'm proud of my work. Ship it! :ship:

### CODE REVIEW

File: aws_topology/tests/cloudformation_infra/sqs-poll.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import sys
```
Comment (vishalxebia) [inline aws_topology/tests/cloudformation_infra/sqs-poll.py:1]: Reason to add this file.

File: aws_topology/stackstate_checks/aws_topology/resources/api_gateway.py [hunk 2]
```diff
@@ -216,13 +216,17 @@ def process_stage(self, data, api, resources, rest_api_arn):
...
             stage.stageName,
         )
-        stage_data = {
-            "DeploymentId": stage.deploymentId,
-            "StageName": stage.stageName,
-            "RestApiId": api.id,
-            "RestApiName": api.name,
-        }
-        stage_data.update(
+        output = make_valid_data(data)
```
[the remark this answers is not in the document]
Comment (Eelviny) [reply aws_topology/stackstate_checks/aws_topology/resources/api_gateway.py:219]: output is the standard name used across all other resources, this change was made for consistency

File: aws_topology/tests/cloudformation_infra/sqs-poll.py (hunk 1 above)
Comment (vishalxebia) [reply aws_topology/tests/cloudformation_infra/sqs-poll.py:1]: @Eelviny Can we document that in Readme? How this file can be useful for testing and how to run it?

Comment (Eelviny) [reply aws_topology/tests/cloudformation_infra/sqs-poll.py:1]: Done!

**Review by vishalxebia — APPROVED** [review]

**Pull request merged**